protocols/kad/CHANGELOG.md

```diff
@@ -10,7 +10,8 @@
   See [PR 4698](https://github.com/libp2p/rust-libp2p/pull/4698).
 - Remove previously deprecated type-aliases.
   Users should follow the convention of importing the `libp2p::kad` module and referring to symbols as `kad::Behaviour` etc.
-  See [PR 4733](https://github.com/libp2p/rust-libp2p/pull/4733). 
+  See [PR 4733](https://github.com/libp2p/rust-libp2p/pull/4733).
```

Comment:
Why is this line a diff?

Reply:
Hmm, must be a whitespace issue, will reset to master